From e631548027cae026486300fe93542949df73a87d Mon Sep 17 00:00:00 2001
Message-Id: <e631548027cae026486300fe93542949df73a87d.1649937186.git.geliang.tang@suse.com>
From: Colin Ian King <colin.king@canonical.com>
Date: Wed, 9 Jun 2021 12:21:28 +0100
Subject: [PATCH] ntb: ntb_pingpong: remove redundant initialization of
 variables msg_data and spad_data
Git-commit: e631548027cae026486300fe93542949df73a87d
Patch-mainline: v5.15-rc1
References: bsn#170
Modified-by-SEL: No


The variables msg_data and spad_data are being initialized with values
that are never read, they are being updated later on. The initializations
are redundant and can be removed.

Addresses-Coverity: ("Unused value")
Signed-off-by: Colin Ian King <colin.king@canonical.com>
Acked-by: Serge Semin <fancer.lancer@gmail.com>
Signed-off-by: Jon Mason <jdmason@kudzu.us>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/ntb/test/ntb_pingpong.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ntb/test/ntb_pingpong.c b/drivers/ntb/test/ntb_pingpong.c
index 2164e8492772..8aeca7914050 100644
--- a/drivers/ntb/test/ntb_pingpong.c
+++ b/drivers/ntb/test/ntb_pingpong.c
@@ -187,7 +187,7 @@ static void pp_ping(struct pp_ctx *pp)
 
 static void pp_pong(struct pp_ctx *pp)
 {
-	u32 msg_data = -1, spad_data = -1;
+	u32 msg_data, spad_data;
 	int pidx = 0;
 
 	/* Read pong data */
-- 
2.34.1

